tests/pages/services/ServiceActions-cy.js

```diff
@@ -220,10 +217,8 @@ describe("Service Actions", function() {
         });
         cy
           .get(".confirm-modal .button-collection .button-danger")
-          .as("dangerButton")
-          .click();
```

Comment:
addressed. The test was obsolete therefore I have adjusted to accommodate the changes in the component